src/plugins/intel_gna/src/frontend/layer_quantizer.hpp

```diff
@@ -31,6 +31,26 @@ InferenceEngine::Precision GetWeightsPrecision(const LayerInfo& layer_info,
                                                const QuantizedLayerParams& quant_layer_params,
                                                const Config& gna_config);
 
```

Comment:
Param is missing in doxygen.
The comment does not explain function designation more than function name, its arguments name and return type. Maybe it make sense to remove it at all. If note please add missing info.

Reply:
Param will be added where missing.